srcpkgs/fail2ban/files/fail2ban/run

```diff
@@ -2,4 +2,4 @@
 exec 2>&1
 mkdir -p /var/run/fail2ban
 
-exec fail2ban-server -f
+exec fail2ban-server -f >/dev/null 2>&1
```

Comment:
Be aware that this _may_ drop any errors or logging output happening from the server.

Reply:
What's the point of this change?

Reply:
> What's the point of this change?

It prevents the spamming the local login with messages every time you change something (Server OK and so on). I can remove it so users van add it themselves if it has other unwanted effects I'm not aware of.